flow/Stats.actor.cpp

```diff
@@ -45,21 +45,32 @@ double Counter::getRate() const {
 }
 
 double Counter::getRoughness() const {
-	double elapsed = now() - interval_start;
+	double elapsed = now() - roughness_interval_start;
```

Comment:
I feel `elapsed` should indeed counted from `interval_start` since `interval_delta` means `n` increments made since `interval_start`, not since `roughness_interval_start` which is when the last event time. Because semantically `elapsed` should equal to `N*mean`.

Reply:
My understanding is wrong. Ignore my comments.